src/sparseml/keras/__init__.py

```diff
@@ -24,9 +24,23 @@
 
     version = [int(v) for v in tensorflow.__version__.split(".")]
     if version[0] != 2 or version[1] < 2:
-        raise Exception
+        raise RuntimeError("TensorFlow >= 2.2 is required, found {}".format(version))
```

Comment:
Why are we still restricted on 2.2 right now?

Reply:
Updated to support from TF 2.1: for TF2.0 there's a bug in TF model save.